exampleSite/config.toml

```diff
@@ -6,15 +6,35 @@ theme = "glam"
 
 [menu]
   [[menu.main]]
-    identifier = 'home'
-    name = 'home'
-    url = '/'
-    weight = -1
+    identifier = 'jobs'
+    name = 'Combat Jobs'
+    url = '/jobs/'
+    weight = 10
   [[menu.main]]
-    identifier = 'about'
-    name = 'about'
-    url = '/about/'
-    weight = 100
+    identifier = 'other-jobs'
+    name = 'Other Jobs'
+    url = '/jobs/'
```

Comment:
Other Jobs links to the same page as Combat Jobs? I'm guessing this is supposed to cover DoH/DoL mostly, will these have their own page?

Reply:
They will, but I was only implementing this in the exampleSite config only to show how the menu bar will look on the final site without implementing all the requisite content.